docs-new/docusaurus.config.js

```diff
@@ -0,0 +1,84 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Lage',
+  tagline: 'A Beautiful JS Monorepo Task Runner',
+  url: 'https://microsoft.github.io',
+  baseUrl: '/',
```

Comment:
pretty sure this has to be lage/docs-new

Reply:
updated the url to -` https://microsoft.github.io/lage` and baseUrl to - `/docs-new`